portal/src/main/webapp/js/src/oncoprint/webgl/src/js/oncoprintmodel.js

```diff
@@ -1012,7 +1049,54 @@ var OncoprintModel = (function () {
     OncoprintModel.prototype.isTrackSortDirectionChangeable = function (track_id) {
 	return this.track_sort_direction_changeable[track_id];
     }
+    
+    OncoprintModel.prototype.isTrackExpandable = function (track_id) {
+	// return true if the flag is defined and true
+	return Boolean(this.track_expandable[track_id]);
```

Comment:
I consider using the `Boolean()` function more explicit and simple than `!!`, even if it is also a constructor (when used as `new Boolean()`) for a class of objects that are truthy (because all objects are).